Firmware/Marlin_main.cpp

```diff
@@ -1116,9 +1116,9 @@ void setup()
 		//disabled filament autoload (PFW360)
 		fsensor_autoload_set(false);
 #endif //FILAMENT_SENSOR
-          // ~ FanCheck -> on
-          if(!(eeprom_read_byte((uint8_t*)EEPROM_FAN_CHECK_ENABLED)))
-               eeprom_update_byte((unsigned char *)EEPROM_FAN_CHECK_ENABLED,true);
+		// ~ FanCheck -> on
+		if(!(eeprom_read_byte((uint8_t*)EEPROM_FAN_CHECK_ENABLED)))
```

Comment:
May be, when you already touched these lines, setting the `EEPROM_FAN_CHECK_ENABLED` to true always in farm mode may save a few bytes as well.
`eeprom_update_byte` checks the EEPROM cell's content and wites a new value only if the previous was different.

Reply:
It saves 12 bytes of flash 👍